packages/nimble-components/performance/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="../dist/all-components-bundle.min.js"></script>
+</head>
+<body>
+    <h1>Wafer Map Performance Tests</h1>
+    <script>
+        document.addEventListener('DOMContentLoaded', main);
+        
+        const colorScales = {
+            colors: ['red', 'orange', 'yellow', 'green'],
+            values: ['1', '33', '66', '100']
+        };
+
+        function createWaferMap(functionName, dieCount) {
+            performance.mark(`${functionName}_start`);
+            const waferMap = document.createElement('nimble-wafer-map');
+            waferMap.dies = generateDies(dieCount).wafermapDieSet;
```

Comment:
can you leave the die data generation outside of the performance measures?

Reply:
sure, now it is outside